web/sites/default/monolog.services.yml

```diff
@@ -1,23 +0,0 @@
-parameters:
```

Comment:
@kxwang, any thoughts on why this yml files was touched?

Reply:
That's an accidental commit. Let me revert it.